src/module.c

```diff
@@ -5746,6 +5749,10 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  *              invoking the command, the error is returned using errno mechanism.
  *              This flag allows to get the error also as an error CallReply with
  *              relevant error message.
+ *     * 'D' -- A "Dry Run" mode.  Return before executing the underlying call().
+ *              if everything succeeded, it will return with a NULL, otherwise it will
```

Comment:
```suggestion
 *     * 'D' -- A "Dry Run" mode. Return before executing the underlying call().
 *              If everything succeeded, it will return with a NULL, otherwise it will
```